src/mode/c_cpp.js

```diff
@@ -5,14 +5,14 @@ var TextMode = require("./text").Mode;
 var c_cppHighlightRules = require("./c_cpp_highlight_rules").c_cppHighlightRules;
 var MatchingBraceOutdent = require("./matching_brace_outdent").MatchingBraceOutdent;
 var Range = require("../range").Range;
-var CstyleBehaviour = require("./behaviour/cstyle").CstyleBehaviour;
+var CstyleBehaviour = require("../mode/behaviour/cstyle").CstyleBehaviour;
 var CStyleFoldMode = require("./folding/cstyle").FoldMode;
 
 var Mode = function() {
     this.HighlightRules = c_cppHighlightRules;
 
     this.$outdent = new MatchingBraceOutdent();
-    this.$behaviour = new CstyleBehaviour();
+    this.$behaviour = new CstyleBehaviour({closeDocComment: true});
```

Comment:
where do we need this to be false? maybe it would be better to make it true by default?

Reply:
I am not sure if this would be the right approach. For example, Python uses `this.$behaviour = this.$defaultBehaviour;`, which is essentially a generic `CStyleBehaviour`. However, Python itself employs a different syntax for docstrings, without using `/**`. It would work as expected because I check if the state is `doc-start`, but I am not sure if making `closeDocComment: true` the default setting is the correct approach in general.